usr/local/www/classes/Form/Section.class.php

```diff
@@ -8,6 +8,7 @@ class Form_Section extends Form_Element
 	public function __construct($title)
 	{
 		$this->_title = $title;
+		$this->addClass('panel panel-default');
```

Comment:
Could you change this to 2 separate addClass calls? Doing it like this works, but it breaks removeClass('panel').